lib/rpmi_transport_shmem.c

```diff
@@ -204,7 +204,7 @@ struct rpmi_transport *rpmi_transport_shmem_create(const char *name,
 		return NULL;
 
 	shtrans->shmem = shmem;
-	shtrans->queue_size = rpmi_shmem_size(shmem) / RPMI_QUEUE_MAX;
+	shtrans->queue_size = rpmi_shmem_size(shmem) / RPMI_QUEUE_MAX / 2;
```

Comment:
This change is not correct.

The shared memory provided to `rpmi_transport_shmem_create()` will be consumed entirely by the 4 queues. If some part of the shared memory is to be used as fast channel then the caller should create `rpmi_shmem` instance accordingly.